CMakeLists.txt

```diff
@@ -229,6 +239,12 @@ if(JEDI_DRIVER)
   list(APPEND _ufs_defs_private JEDI_DRIVER=ON)
 endif()
 
+if(GOCART)
+  add_dependencies(ufs UFS)
+  list(APPEND _ufs_defs_private FRONT_GOCART=Aerosol_Cap)
```

Comment:
Why is 'Aerosol_Cap' used to name gocart cap? In all other components the name of the cap is typically named using the name of the component. For example, MOM6 cap is named mom_cap_mod not Ocean_Cap.

Reply:
The UFS component will be named UFS-Aerosols (as per GEFS-Aerosols). Here, Aerosol_Cap is used to distinguish the UFS-Aerosols NUOPC cap from NASA's GOCART component. The UFS-Aerosols cap is a top-level component embedding the MAPL driver, which runs GOCART and other subcomponents, and exchanging prognostic and diagnostic aerosol data with other UFS coupled components.